Assets/Scripts/Model/Timing/AccessCard.cs

```diff
@@ -51,20 +51,7 @@ async private Task Trash()
 
         async private Task Steal()
         {
-            if (card.Stealable != Stealable.CANNOT_STEAL)
-            {
-                card.FlipFaceUp();
-                game.runner.zones.score.Add(card, 2);
-                UnityEngine.Debug.Log("Stole CST!");
-
-                /**
-                    Maybe each card can respond to being stolen:
-                    - you must steal me (Project Atlas)
-                    - you can steal me if you pay me (Obokata Protocol)
-                    - you cannot steal me (asset, operation, Lakshmi Smartfabrics, Old Hollywood Grid, Haarpsichord, Trebuchet etc.)
```

Comment:
Done for assets and operations, but not for the special"cannot steal" effects. I think the easiest to write is Haarpsichord, because it doesn't require Corp AI to actively use it.

- [ ] Haarpsichord

Reply:
It makes sense